src/pages/Quiz/Quiz.js

```diff
@@ -8,20 +8,36 @@ import head from '../../public/img/RicksHead.png';
  */
 // eslint-disable-next-line no-unused-vars
 function Quiz(props) {
+  const allAnswers = [];
+  const { element: counterElement, increment, getCount } = QuestionCounter();
+  const onClick = () => {
+    // eslint-disable-next-line no-unused-vars
+    const answeredQuestions = getCount() - 1;
+    // eslint-disable-next-line no-unused-vars
+    const points = allAnswers.filter((answer) => answer.correct).length;
+  };
   // @ts-ignore
-  const { question } = Question(props.generator.next().value);
+  let question = Question(props.generator.next().value);
+
   return html`<div class="${styles.wrapper}">
-  ${BackgroundDecoration()}
+    ${BackgroundDecoration()}
     <div class="${styles.quizContainer}">
-      <div class="${styles.headImgWrapper}"">
+      <div class="${styles.counterElementWrapper}">${counterElement}</div>
+      <div class="${styles.popupCloseWrapper}">${PopupClose({ onClick })}</div>
+      <div class="${styles.headImgWrapper}">
         <img src="${head}" alt="Ricks head" />
       </div>
-      ${question}
+      ${question.question}
       ${Button({
         onClick: (e) => {
           e.preventDefault();
+          allAnswers.push(question.getFullAnswer());
+          // @ts-ignore
+          question = Question(props.generator.next().value);
+          increment();
+          document.getElementById('question').replaceWith(question.question);
```

Comment:
Mamy na to specjalną funkcję od Huberta:
```JS
render({ on: document.getElementById('question'), element: question.question });
```

W tym przypadku chyba nie ma różnicy także zostawiam bardziej jako ciekawostkę.